.spectral.yaml

```diff
@@ -107,4 +107,11 @@ rules:
     then:
       function: length
       functionOptions:
-        max: 95
\ No newline at end of file
+        max: 95
+
+  must-not-use-base-server-url:
+    description: "Must not use base server url in spec"
```

Comment:
Could add the why: to be able to generate a single client gen compatible for multi environment